libraries/AP_HAL_ChibiOS/hwdef/AET-H743-Basic/hwdef-bl.dat

```diff
@@ -0,0 +1,49 @@
+# hw definition file for processing by chibios_pins.py
+# for AET H743-Basic bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 2024
```

Comment:
You should use the logical id here

Reply:
What is exactly "logical id"? Does that mean I should choose an id higher than 7100 in board_types.txt?

Reply:
see https://github.com/ArduPilot/ardupilot/blob/be5c68d74db69dd2f3e5226ff576e1b9b8166c79/libraries/AP_HAL_ChibiOS/hwdef/X-MAV-AP-H743v2/hwdef.dat#L8 and its entry here https://github.com/ArduPilot/ardupilot/blob/be5c68d74db69dd2f3e5226ff576e1b9b8166c79/Tools/AP_Bootloader/board_types.txt#L292